searchlib/src/vespa/searchlib/transactionlog/chunks.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "ichunk.h"
+#include <vespa/vespalib/util/compressionconfig.h>
+
+namespace search::transactionlog {
+
+/// Current default chunk serialisation format
+class XXH64None : public IChunk {
+protected:
+    Encoding onEncode(nbostream &os) const override;
+    void onDecode(nbostream &is) override;
+public:
+};
+
+/// TODO Legacy chunk serialisation format to be removed soon.
+class CCITTCRC32None : public IChunk {
```

Comment:
Consider adding Chunk suffix to class name.

Reply:
Fixed